tests/cases/with_open.py

```diff
@@ -1,12 +1,14 @@
 #!/usr/bin/env python3
 
+import os
 from tempfile import NamedTemporaryFile
 
 
 if __name__ == "__main__":
-    with NamedTemporaryFile(mode="a+") as temp_file:
+    with NamedTemporaryFile(mode="a+", delete=False) as temp_file:
         file_path = temp_file.name
         with open(file_path, "w") as f:
             f.write("hello")
         with open(file_path, "r") as f:
             print(f.read())
+        os.unlink(file_path)
```

Comment:
try moving this out of the outer `with`.

Reply:
Then `file_path` goes out of scope.  Looks like I have to make a more invasive change that also fixes https://github.com/adsharma/py2many/issues/298

Reply:
It goes out of scope in Rust?

If that is it, wrap the unlink in a try block and ignore deleting the temporary file on Windows.  temp files are a special case, and windows is an even more special case ;-)

Or, instead of using temporary files for the test case, use a generated filename that is unique for each language, which can be supplied in the program args using [`CASE_ARGS`](https://github.com/adsharma/py2many/blob/main/tests/test_cli.py#L48), like `$TEMP/py2many-with-open-$lang.txt`.

Reply:
I did a temporary file with delete=False which is logically equivalent to generating a unique filename for each language.